### PR TITLE
fix: add new bill statuses

### PR DESCRIPTION
**Pull request by elseagle**
We need to add reported-out-of-committee and concurrence to the validator checker which is used in the data-curation repo to validate manually curated bills

### CODE REVIEW

**Comment by johnseekins** [comment]
Don't forget to bump the version in `pyproject.toml` so we can release this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 6.16.6 - August 9, 2023
+
+* Add new bill statuses to validation
+
 ## 6.16.5 - May 16, 2023
```
[the remark this answers is not in the document]
Comment (johnseekins) [reply CHANGELOG.md:7]: I forgot to update it for the last few releases. Version _should_ be 6.17.5, not 6.16.6.

[the remark this answers is not in the document]
Comment (johnseekins) [reply CHANGELOG.md:7]:
```
## 6.17.4 - August 2, 2023

* Small territory fixes

## 6.17.3 - August 1, 2023

* update pyyaml dependency
      
## 6.17.2 - August 1, 2023

* Support additional territories in db init
      
## 6.17.0 - July 14, 2023

* Add additional US territory support#113
```
All other releases were me messing up tagging. Now that we can actually do a proper [release](https://github.com/openstates/openstates-core/releases) that shouldn't happen again.

[the remark this answers is not in the document]
Comment (elseagle) [reply CHANGELOG.md:7]: Roger that

**Pull request merged**